### DISCUSSION TITLE
no-duplicate-imports

### DISCUSSION THREAD

**Discussion by aselcuktuncer** [biomejs/biome#2063] [Ideas]
I checked the doc but could not find this rule. I therefore propose that the following rule be added: https://eslint.org/docs/latest/rules/no-duplicate-imports

Example:

```
import { pgEnum, pgTable, primaryKey, smallint, text, uuid, varchar } from 'drizzle-orm/pg-core';
import { numeric } from 'drizzle-orm/pg-core';
import { boolean } from 'drizzle-orm/pg-core';
```
--->

`import { pgEnum, pgTable, primaryKey, smallint, text, uuid, varchar, numeric, boolean } from 'drizzle-orm/pg-core';`

**Comment by ematipico** [comment]
Closing. Please use https://github.com/biomejs/biome/discussions/3, that's where we collect feedback/ideas/etc.
